packages/calcite-components/src/components/input-number/input-number.e2e.ts

```diff
@@ -923,6 +915,26 @@ describe("calcite-input-number", () => {
     }
   });
 
+  // refer to issue here https://github.com/Esri/calcite-components/issues/6854
```

Comment:
Can you add the issue number in the test name instead? Similar to https://github.com/Esri/calcite-components/blob/14036c1354419222bc5b839af8b68675b17ebf62/packages/calcite-components/src/components/tree/tree.e2e.ts#L962.